datacenter/dtr/2.3/guides/release-notes.md

```diff
@@ -10,19 +11,43 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](admin/upgrade.md),
 to upgrade your installation to the latest release.
 
+## DTR 2.3.1
+
+(24 August 2017)
+
+### Bugs fixed
+
+* High severity:
+  * Fixed a bug, causing upgrades to fail when upgrading from 2.1 to 2.2, and
+  then to 2.3.
+  * Make it possible to install DTR when the Docker daemon has SELinux enabled.
+* Low severity:
+  * In-product documentation for content cache now show a simplified
+  configuration format. The older configuration format is still supported for
+  backwards compatibility.
+  * Improved the reliability of the DTR installer.
```

Comment:
Just changed order of operations around a bit so that killing the installer leads to better failures. It doesn't really matter. We can omit this.